capabilities/writetarget/monitoring/pb/platform/on-chain/forwarder/report_processed.proto

```diff
@@ -0,0 +1,52 @@
+syntax="proto3";
+
+package platform.on_chain.forwarder;
+option go_package = ".;forwarder";
+
+// The on-chain ReportProcessed event which is extracted from the tx/event data or
+// the write-target WriteConfirmed event, after a write was confirmed (@see message: platform.write-target.WriteConfirmed).
+message ReportProcessed {
+  // Event data
+  string receiver = 1;
+  string workflow_execution_id = 2; // bytes as hex string for readability
+  uint32 report_id = 3;
+  bool success = 4;
+
+  // Head data - when was the event produced on-chain
+  string block_hash = 5;
+  string block_height = 6;
+  uint64 block_timestamp = 7;
+
+  // Transaction data - info about the tx that mained the event (optional)
+  string tx_id = 10; // TXM ref
+  string tx_hash = 11;
+  string tx_sender = 12;
+  string tx_receiver = 13;
+  string tx_status = 14;
+
+  // [Execution Context]
+  // TODO: replace with a proto reference once supported
+  // Execution Context - Source
+  string meta_source_id = 20;
```

Comment:
afaik this is now available and we can use nested protos so the work to extract this as a separate Proto type can be scheduled

Reply:
done